packages/lit-dev-content/site/playground/signin.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="robots" content="noindex" />
```

Comment:
Nit: no closing required `/>`. Can be `>`

Reply:
Done.